lib/ansible/modules/cloud/vmware/vmware_guest_disk.py

```diff
@@ -108,6 +108,20 @@
      - '   If C(state) is set to C(absent), disk will be removed permanently from virtual machine configuration and from VMware storage.'
      - '   If C(state) is set to C(present), disk will be added if not present at given SCSI Controller and Unit Number.'
      - '   If C(state) is set to C(present) and disk exists with different size, disk size is increased.'
+     - ' - C(iolimit) is set, disk will be added with the values specified under iolimit section: Valid values are:'
+     - '     - C(limit)(long) limit value to be given, by default it goes to custom.'
+     - '     - C(shares) If this section is set disk will be added with the values specified under shares section: Valid values are:'
+     - '         - C(level)(string): Valid values are:'
+     - '              - C(low) low level'
+     - '              - C(normal) noraml level'
+     - '              - C(High) High level'
+     - '              - C(custom) custom level'
+     - ' - C(shares) If this section is set disk will be added with the values specified under shares section: Valid values are:'
```

Comment:
`levelValue` is not documented. Also, name parameter as `level_value` to match other parameter names.